Source/WebKit/Platform/cocoa/WebPrivacyHelpers.mm

```diff
@@ -302,6 +302,7 @@ void requestLinkDecorationFilteringData(LinkFilteringRulesCallback&& callback)
 {
     m_cachedQuirks = WTFMove(quirks);
     m_cachedQuirks.shrinkToFit();
+    RELEASE_LOG_ERROR(ResourceLoadStatistics, "StorageAccessPromptQuirkController::setCachedQuirks: Loaded %lu storage access prompt quirks from WebPrivacy.", quirks.size());
```

Comment:
(Extremely, extremely minor) nit — `(s)` here, so the `1` case reads a bit more nicely 😅

```suggestion
    RELEASE_LOG_ERROR(ResourceLoadStatistics, "StorageAccessPromptQuirkController::setCachedQuirks: Loaded %lu storage access prompt quirk(s) from WebPrivacy.", quirks.size());
```